code/api/views.py

```diff
@@ -229,10 +229,18 @@ def post(self, request, author_id):
 @method_decorator(csrf_exempt, name='dispatch')
 class PostView(View):
 
+    # @method_decorator(authenticate_request)   Can we have this too?
+    @method_decorator(validate_node)
```

Comment:
Not really a feedback comment, but not sure if we want this route to be node_authenticated? I'm not sure what the requirement is. Should the other GET routes be validated? Maybe @JawadHossain could give his thoughts.

Reply:
I'm a bit confused about which endpoint has to be validated, but I think we can get rid of it, since it's for a public post?

Reply:
Only the inbox post route should use validate_node. 
We need to authenticate users instead when we use the api to edit posts for example.

Reply:
Ok, sounds good
Will make the change